### PR TITLE
#167377527 Setup Eslint

### PR DESCRIPTION
**Pull request by ibkadeeko**
**What does this PR do?**
This pull request sets up code linting using eslint

**Description of Task to be completed**
- Install `eslint`, `eslint-config-airbnb` and `eslint-plugin-import`
- Add eslint config

**How should this be manually tested?**
N/A

**Any Background Context You want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
[#167377527](https://pivotaltracker.com/story/show/167377527)

**Screenshots (if appropriate)**
N/A

**Questions**
N/A

### CODE REVIEW

**Review by Nennyfills — DISMISSED** [review]
Nice work!

**Review by Jonathan4github — DISMISSED** [review]
Great job

**Review by Merkll — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -9,22 +9,11 @@
   "author": "Andela Simulations Programme",
   "license": "MIT",
   "dependencies": {
-    "body-parser": "^1.18.3",
```
Comment (iverenshaguy) [inline package.json]: Kindly replace these packages as they do not fall under setting up eslint.

**Review by iverenshaguy — CHANGES_REQUESTED** [review]
Good job. Please review my comments.

File: package.json (hunk 1 above)
Comment (ibkadeeko) [reply package.json]: Ok.
Will do that.

**Comment by iverenshaguy** [comment]
I have a few comments about your PR. The commit message format is
```
chore/feat/fix(<scope>): add commit message format

- add the right format
- add the right format again

[Starts/Finishes/Fixes #1234567890]
```
The only thing that should start with a capital letter in your commit message is the footer i.e `Finishes/Starts/Fixes` or class name. 

Also, the PR message format is

**What does this PR do?**
This pull request sets up code linting using eslint

**Description of Task to be completed**
- Install `eslint`, `eslint-config-airbnb` and `eslint-plugin-import`
- Add eslint config

**How should this be manually tested?**
N/A

**Any Background Context You want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
[#167377527](https://www.pivotaltracker.com/story/show/167377527)

**Screenshots (if appropriate)**
N/A

**Questions**
N/A

Your pull request description should be professional and as descriptive as possible and your story ID should link to the story URL for easy access.

**Comment by ibkadeeko** [comment]
Thank you.
Noted

**Comment by iverenshaguy** [comment]
Good job. However, your commit message needs a few corrections 
```
chore(linting): set up code linting using eslint

- install eslint, eslint-plugin-import and eslint-config-airbnb
- add eslint config

[Starts #167377527]
```

The new line after the commit header is really important and there shouldn't be a slash after chore. Also, please pay more attention to details in your commit message, there's a typo `plugion`

**Comment by ibkadeeko** [comment]
Ok
Thanks for the feedback

**Review by iverenshaguy — APPROVED** [review]
LGTM

**Comment by ibkadeeko** [comment]
Ok

**Pull request merged**